utils/helpers.py

```diff
@@ -54,7 +100,12 @@ def get_assets(
         }
         for a in assets
     ]
-    assets = [a for a in assets if len(a["labels"]) > 0]
+    if not get_labeled and not get_unlabeled:
+        raise ValueError("no label types selected")
+    if not get_labeled:
+        assets = [a for a in assets if len(a["labels"]) > 0]
```

Comment:
no labeled means that we keep the labeled (so the opposite of what is written.
To make the code less confusing and more efficient, you can create a function `is_kept(asset, get_labeled, get_unlabeled)`, and filter your list depending on the results (walk the list only once)